subscribers/logs/logger.tests/LoggerTests.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using Xunit;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json;
+using Microsoft.Data.Sqlite;
+using Dta.Marketplace.Subscribers.Logger.Worker;
+using Amazon.SQS.Model;
+
+using Microsoft.EntityFrameworkCore;
+
+namespace Dta.Marketplace.Subscribers.Logger.Worker.Logger.Tests {
+
+    public class LoggerTests {
+
+        [Fact]
+        public void Can_log_information_message() {
+            // Arrange
+            var logger = new Mock<ILoggerAdapter<AppService>>();
+            var dbSet = new Mock<DbSet<LogEntry>>();
+            var connection = new SqliteConnection("DataSource=:memory:");
+            connection.Open();

```

Comment:
need to close the connection also. checkout https://docs.microsoft.com/en-us/ef/core/miscellaneous/testing/sqlite
`
try {
CODE
} finally {
connection.close()
}
`

Reply:
done